src/nanorc/cfgsvr.py

```diff
@@ -79,7 +96,74 @@ def save_on_resume(self, data: dict) -> dict:
         f.write(json.dumps(data, indent=2))
         f.close()
 
-
+    
+    def save_on_stop(self, run:int):
+        pass
+        
+
+        
+
+class DBConfigSaver(SimpleConfigSaver):
```

Comment:
Sorry, I've just noticed this now.
Why `DBConfigSaver` inherits from `SimpleConfigSaver`.
The two share the interface but only the `SimpleConfigSaver` writes to disk.
I would suggest to either drop inheritance or to create a common base class.

As bonus, the `DBConfigSaver` `cfg_outdir` argument will not be needed anylonger.